Sources/PointFree/Account/Account.swift

```diff
@@ -32,106 +32,73 @@ private func fetchAccountData<I>(
 ) -> IO<Conn<I, AccountData>> {
   @Dependency(\.database) var database
   @Dependency(\.stripe) var stripe
+  @Dependency(\.fireAndForget) var fireAndForget
 
   let (user, subscriberState) = lower(conn.data)
 
-  let subscription = EitherIO {
-    try await database.fetchSubscription(user: user)
-  }
-
-  let owner =
-    subscription
-    .flatMap { subscription in
-      EitherIO { try await database.fetchUser(id: subscription.userId) }
+  return IO { () -> AccountData in
```

Comment:
I refactored the internals of `fetchAccountData` to use async/await, but didn't refactor the signature of it to be modern because it's still plugged into a legacy thing.